pymemcache/client/base.py

```diff
@@ -254,6 +269,13 @@ def __init__(self,
         self.key_prefix = key_prefix
         self.default_noreply = default_noreply
         self.allow_unicode_keys = allow_unicode_keys
+        self.flush_on_reconnect = flush_on_reconnect
+        self.flush_on_next_connect = False
+        if self.flush_on_reconnect:
+            logger.warn('You start client with `flush_on_reconnect` activated '
```

Comment:
I'd prefer to note repeatedly warn users, and rather trust that they have read the docs and know their business.